apps/frontend-manage/src/components/courses/MicroLearningTile.tsx

```diff
@@ -28,48 +28,48 @@ import { useRouter } from 'next/router'
 import { useState } from 'react'
 import { twMerge } from 'tailwind-merge'
 import StatusTag from './StatusTag'
-import MicroSessionDeletionModal from './modals/MicroSessionDeletionModal'
+import MicroLearningDeletionModal from './modals/MicroLearningDeletionModal'
 import PublishConfirmationModal from './modals/PublishConfirmationModal'
 
-interface MicroSessionProps {
-  microSession: Partial<MicroSession> & Pick<MicroSession, 'id' | 'name'>
+interface MicroLearningProps {
+  microLearning: Partial<MicroLearning> & Pick<MicroLearning, 'id' | 'name'>
 }
 
-function MicroSessionTile({ microSession }: MicroSessionProps) {
+function MicroLearningTile({ microLearning }: MicroLearningProps) {
   const t = useTranslations()
   const router = useRouter()
 
   const [publishModal, setPublishModal] = useState(false)
   const [copyToast, setCopyToast] = useState(false)
   const [deletionModal, setDeletionModal] = useState(false)
 
-  const [unpublishMicroSession] = useMutation(UnpublishMicroSessionDocument, {
-    variables: { id: microSession.id },
+  const [unpublishMicroLearning] = useMutation(UnpublishMicroLearningDocument, {
+    variables: { id: microLearning.id },
   })
 
-  const href = `${process.env.NEXT_PUBLIC_PWA_URL}/micro/${microSession.id}/`
+  const href = `${process.env.NEXT_PUBLIC_PWA_URL}/microlearning/${microLearning.id}/`
 
-  const isFuture = dayjs(microSession.scheduledStartAt).isAfter(dayjs())
-  const isPast = dayjs(microSession.scheduledEndAt).isBefore(dayjs())
+  const isFuture = dayjs(microLearning.scheduledStartAt).isAfter(dayjs())
+  const isPast = dayjs(microLearning.scheduledEndAt).isBefore(dayjs())
 
   return (
     <div
       className="p-2 border border-solid rounded w-full sm:min-w-[18rem] sm:max-w-[18rem] border-uzh-grey-80"
-      data-cy={`microlearning-${microSession.name}`}
+      data-cy={`microlearning-${microLearning.name}`}
     >
       <div className="flex flex-row items-center justify-between">
         <Ellipsis maxLength={25} className={{ markdown: 'font-bold' }}>
-          {microSession.name || ''}
+          {microLearning.name || ''}
         </Ellipsis>
 
-        {microSession.status === MicroSessionStatus.Draft && (
+        {microLearning.status === PublicationStatus.Draft && (
```

Comment:
Currently, the database uses a separate `MicroLearningStatus` instead of the `PublicationStatus`. A corresponding GraphQL type is available as well. Should we change this here?